src/api/youtube.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+
+// const KEY ='AIzaSyBmctwBX84fmmweUPTCeiCJ87omLGGWB30';
+// const KEY ='AIzaSyDCaoflqhPGGQgBAp82mVmiTQLzsJPAcmE';
+const KEY = 'AIzaSyC4NmD6_LsTwuYW6eqmCgf46kng-Q4OkVs'
```

Comment:
@RajanTank move this key into the separate file called config.js

Reply:
Ideally it should come from env variables, find how to use env files in node.